app/retail/emails.py

```diff
@@ -471,43 +471,43 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from external_bounties.models import ExternalBounty
-    subject = "Getting Started With Gitcoin | Gitcoin Profiles"
+    subject = "Gitcoin Q2 Update | Augur Testimonial"
 
     intro = '''
 
 <p>
     Hi there
 </p>
 <p>
-This week, we shipped <a href="https://medium.com/gitcoin/getting-started-with-gitcoin-fa7149f2461a">
-Getting Started With Gitcoin.</a>
-In this post designed for our Gitcoin contributors, you’ll find all the info you
-need to set up your Gitcoin account and get to work contributing to open source projects!
+This week, we shipped <a href="https://medium.com/gitcoin/gitcoin-q2-update-c31e751889c8">Q2 updates.</a>
+In this update, we discuss our OKR’s for Q2, how we performed, and our goals moving forward into Q3.
 </p>
-We also published our first installment of
-<a href="https://medium.com/gitcoin/gitcoiner-profile-kenneth-ashley-b8f6e8b458a6">Gitcoin Profiles.</a>
-Gitcoiner Profiles are our way of saying thank you to our great community of open source developers for
-all of their hard work.
-This week's profile features Kenneth Ashley who has worked with Market Protocol, MyCrypto, CodeFund, and Gitcoin.
 
+<p>
+In addition to that, we highlighted <a href="https://medium.com/gitcoin/gitcoin-testimonials-augur-9bfe97368a30">
+Augur's work on the Gitcoin platform.</a>
+In this post, we briefly discuss what the goal of the Augur project is, how the team has used Gitcoin
+to build it, and some specifics about the important bounties they’ve had completed by the Gitcoin community.
+Special shout out to Tom Kysar and @cryptomental for their input on this one!
+</p>
+
+<p>
+Finally, we pushed our
+<a href="https://medium.com/p/804c18dc91da/edit">second installment of Gitcoiner profiles.</a>
```

Comment:
edit link - should be public link

Reply:
needs image http://bits.owocki.com/0L0Q0n2w3T0v/Screen%20Shot%202018-07-05%20at%206.31.07%20PM.png